src/content/docs/tutorial-app-performance/guide-to-monitoring-core-web-vitals.mdx

```diff
@@ -202,14 +202,23 @@ Let's say your organization recently updated images on the site’s home page on
     </Collapser>
 </CollapserGroup>
 
-## What's next? [#next]
+## Get more context with session traces [#session-traces]
+
+New Relic captures data about a user's session on a given page. Session traces are randomly sampled and recorded in New Relic, letting you view how a user experienced page load timing issues, JavaScript events, or other errors. 
+
+<img
+    title="Session traces in the New Relic UI"
+    alt="Session traces in Browser UI lets you look at real user sessions for a given page"
+    src={browserSessionTraces}
+/>
 
-This tutorial covered some actions you can take with your core web vitals. You have some options for your next steps.
+We recommend using session traces to find patterns across different user sessions. If one of your core web vitals scores could be improved, one option is to look at session traces to see what common attributes are shared across different user sessions. Here's an example of how session traces can help you improve your site:
 
-* If you're noticing missing data in **Web vitals**, then check out [Missing data on the Web vitals page](/docs/browser/new-relic-browser/troubleshooting/missing-data-on-web-vitals-page) for some solutions.
-* If you want to go deeper with core web vitals, you can learn more with our free [self-paced core web vitals training course](https://learn.newrelic.com/self-paced-core-web-vitals/).
+1. From the **Web vitals** tab, click the page that's performing poorly. Using the above image, you see that your site has a good overall LCP score, but you also notice some of your individual pages need improvement. 
+2. Looking at the **Page URLs** section of the **Web vitals** page, you see that your homepage URL has an LCP score of greater than four seconds. Is this due to an error that affects all users, or is it only affecting one kind of user? For example, do you notice that users with one version of Chrome or a certain operating system are experiencing slow page load times? 
```

Comment:
Instead of directly asking the user the question in the second sentence, I would probably rephrase this to something along the lines of "You'll need to find out if this is caused by an error that affects all users,...". Again, just a small suggestion